docker/README.rst

```diff
@@ -1,69 +1,152 @@
 Docker Deployment
 ====================================
 
+.. TODO
+
+    Docs to update:
+
+    Jobe thing turns red?
+
 .. note::
 
-    These instructions are for installing Runestone Server using Docker to automate
+    These instructions are for installing the Runestone servers using Docker to automate
     much of the work involved in setting up the environment needed to run a server.
     If you would prefer to install all the components directly on a server yourself,
     see the `Manual Installation instructions <../docs/installation.html>`_.
 
 Using Docker, we can bring up the server without needing to install dependencies directly on
 the host. Instead, the software required to run the server will be installed in four separate
-Docker containers (the Runestone application, redis cache, postgres DB, and Jobe code compilation).
-
-.. note::
-
-    These instructions have been tested on Ubuntu 20.04 and should work on any version of Ubuntu 18.04+.
-    Installation on older versions of Ubuntu or other Linux distributions may require adjustments.
-    These instructions have also been tested on OS X. For Windows, use `WSL2 <https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-10#1-overview>`_, VirtualBox, or other similar virtualization software.
+Docker containers (the Runestone application, redis cache, postgres DB, and Jobe code runner).
 
 
 **Contents**
 
 .. contents::
     :local:
+    :depth: 2
 
 
 Setup
 -----------------------------
+To build a Docker application with the server and all its dependencies:
+
+1. Install OS-dependent prerequisites
+*************************************
+
+.. TODO
+
+    Enable the script to download and install the Docker Desktop if necessary. Challenge: how to determine if an app in installed in Windows/OS X? On Windows, should we assume WSL2 + Docker Desktop or ask the user if they want to VirtualBox? (I think we should assume -- an advanced user would probably understand to install VirtualBox if they wanted to.)
```

Comment:
the pretext script has a little function that attempts to run a given program and will inform you when you are missing a prerequisite.

I think that would be very user friendly -- we know the prereqs lets make sure they are all there up front and let the user know they need to install them before going any further.

Reply:
Per our verbal discussion, we resolved this.